protocol/0044-LIME-lp_mechanics.md

```diff
@@ -334,18 +339,40 @@ In the case of spot markets it will be transferred into the network treasury for
+- For a market that is in continuous trading and LP has committed liquidity
+  - when `market.liquidity.providers.fee.calculationTimeStep` is set to `0` any funds that are in `ACCOUNT_TYPE_FEES_LIQUIDITY` account will be distributed to `ACCOUNT_TYPE_LP_LIQUIDITY_FEES` on the next block (<a name="0044-LIME-061" href="#0044-LIME-061">0044-LIME-061</a>)
+  - if `market.liquidity.providers.fee.calculationTimeStep` is set to `10s` and `validators.epoch.length` is set to `15s`, during the first `10` seconds of the current epoch parameter change `market.liquidity.providers.fee.calculationTimeStep = 3s` is enacted, at the end of the epoch any funds that are in `ACCOUNT_TYPE_FEES_LIQUIDITY` account will be distributed to `ACCOUNT_TYPE_LP_LIQUIDITY_FEES` on the next block. For the next epoch the distribution will take place at `3` second intervals (<a name="0044-LIME-062" href="#0044-LIME-062">0044-LIME-062</a>)
+- For a market that is in continuous trading if a new LP has active buy and sell orders on the market then makes a liquidity commitment to that market, at the start of the next epoch the active orders will count towards the LPs liquidity commitment. (<a name="0044-LIME-090" href="#0044-LIME-090">0044-LIME-090</a>)
+- If an LP with a liquidity provision and active orders on a market cancel their provision only (orders remain active), after the cancellation penalty (if it applies at end of epoch) the end of next epoch lp will not accrue any rewards or incur penalty for trades on the market. (<a name="0044-LIME-097" href="#0044-LIME-097">0044-LIME-097</a>)
+- In a market where the market parameter is `market.liquidity.priceRange = 0.05 (5%)` and is engaged in continuous trading with a mid price set at 5, a new Liquidity Provider (LP) has infused liquidity by placing a buy order at a price of 4.75 and a sell order at a price of 5.25 (with sufficient volume). As the epoch progresses, if the market parameter is altered to `market.liquidity.priceRange = 0.01` (1%), then upon the culmination of the current epoch, the LP will still have fulfilled their committed notional volume, rendering them exempt from a bond penalty. (<a name="0044-LIME-091" href="#0044-LIME-091">0044-LIME-091</a>)
+
+- In a market, where the market parameter is `market.liquidity.priceRange = 0.05`, and it is engaged in continuous trading with a mid price set at 5, a new Liquidity Provider (LP) has injected liquidity by placing a buy order at a price of 4.74 and a sell order at a price of 5.25 (with sufficient volume). As the epoch unfolds, if the market parameter is modified to `market.liquidity.priceRange = 0.01` (1%), then at the conclusion of the ongoing epoch, the LP will not have met their committed notional volume, resulting in the imposition of a bond penalty. (<a name="0044-LIME-093" href="#0044-LIME-093">0044-LIME-093</a>)
```

Comment:
Please modify this in line with the edits above, for clarity / consistency.

Reply:
done

> - Consider a market, where `market.liquidity.priceRange = 0.05`, and which is in continuous trading with a mid price set at 5. There is an LP who's committed to provide liquidity. They place a buy order at a price of 4.74 and a sell order at a price of 5.25 (with sufficient volume). As the epoch progresses, if the market parameter is altered to `market.liquidity.priceRange = 0.01` (1%), then upon the culmination of the ongoing epoch, the LP will not have met their committed notional volume, resulting in the imposition of a bond penalty. (<a name="0044-LIME-093" href="#0044-LIME-093">0044-LIME-093</a>)